qa/L0_request_cancellation/grpc_cancellation_test.py

```diff
@@ -169,28 +170,61 @@ def test_grpc_async_infer_cancellation_at_step_start(self):
         with open(server_log_name, "r") as f:
             server_log = f.read()
 
-        cancel_at_start_count = len(
```

Comment:
Move the counting for "Cancellation notification received" to test.sh.
Expect one "Cancellation notification received" in log per cancellation after the change to grpc infer_handler.
cc @oandreeva-nv

Reply:
ould you please briefly explain why this move is needed?

Reply:
After the change, the message "Cancellation notification received" is logged one time, which is consistent with other tests. There is an existing check for this message in test.sh already but it does not check the number of occurances. So I just modify the check in `test.sh` to also check for number of occurances.
https://github.com/triton-inference-server/server/blob/cd92b058ec99522cd61bdd4368eade0d8b556891/qa/L0_request_cancellation/test.sh#L117-L125